projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/_all-sites-menu.scss

```diff
@@ -0,0 +1,40 @@
+/**
+ * Styles for All Sites menu
```

Comment:
@okmttdhr looks like we forgot to include this file for Core color schemes, or make this inline CSS. With Core color schemes, this CSS is not applied at all.

Reply:
The masterbar module is not active behind the nav redesign, so we have to put it in other places and use another way to load the styles 😓

Reply:
It would be better to move it into wpcom-site-menu folder and enqueue styles from there

Reply:
I consolidated and enqueued the styles in https://github.com/Automattic/jetpack/pull/36632/commits/0e42678ea7845887c2593395acb0afc925e938d2.
Thank you for your comments!